From 81f7829d0b8d115214d46e9093f3c157f9fd046b Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Mon, 30 Mar 2020 17:44:08 +0200
Subject: [PATCH] Fix creation of $SPACE with GNU Make 4.3

GNU Make 4.3 has this incompatible change:
  * WARNING: Backward-incompatibility!
    Previously appending using '+=' to an empty variable would result in a
    value starting with a space
This is eactly what the code relied on to before.

https://bugzilla.opensuse.org/show_bug.cgi?id=1168051
---
 make/common_variables.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/make/common_variables.mk b/make/common_variables.mk
index 3eabc3a5..8fe07d16 100644
--- a/make/common_variables.mk
+++ b/make/common_variables.mk
@@ -46,8 +46,8 @@ endif
 # a space is needed, since it is not possible to replace a literal
 # space (same goes for comma)
 #
-SPACE :=
-SPACE +=
+empty :=
+SPACE := $(empty) $(empty)
 
 COMMA := ,
 

